plugins/LadspaEffect/calf/src/synth.cpp

```diff
@@ -25,7 +25,7 @@ using namespace std;
 
 void basic_synth::kill_note(int note, int vel, bool just_one)
 {
-    for (list<dsp::voice *>::iterator it = active_voices.begin(); it != active_voices.end(); it++) {
```

Comment:
Hmm... @Lukas-W @diizy Are these old calf plugins our problem now?  I see some tweaks are being made, but upstream has diverged too much in this case, right?